alts/worker/runners/base.py

```diff
@@ -367,16 +368,21 @@ def publish_artifacts_to_storage(self):
         # Should upload artifacts from artifacts directory to preferred
         # artifacts storage (S3, Minio, etc.)
 
+        def replace_host_name(log_string) -> str:
+            return re.sub(r'\[local\]', f'[{self._task_id}', log_string)
+
         def write_to_file(file_base_name: str, artifacts_section: dict):
             log_file_path = os.path.join(
-                self._artifacts_dir, f'{file_base_name}_{self._task_id}.log')
+                self._artifacts_dir,
+                f'{file_base_name}_{self._task_id}_{int(time.time())}.log'
+            )
             with open(log_file_path, 'w+t') as f:
```

Comment:
jfyi
Path from `pathlib` is very useful in similar cases:
for joining a path's parts
```
Path(part1).joinpath(part2)
```
for opening/writing to a file
```
with Path(path_to_file, mode='w+t') as fd:
  fd.write(some_string)
```